src/views/FarmAuction/components/AuctionLeaderboard/AuctionLeaderboardTable.tsx

```diff
@@ -0,0 +1,224 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import BigNumber from 'bignumber.js'
+import {
+  Text,
+  Flex,
+  Box,
+  OpenNewIcon,
+  BunnyPlaceholderIcon,
+  Skeleton,
+  Button,
+  useTooltip,
+  useMatchBreakpoints,
+  useModal,
+} from '@pancakeswap/uikit'
+import { getBscScanLink } from 'utils'
+import { getBalanceNumber } from 'utils/formatBalance'
+import EllipsisMenu, { OptionProps } from 'components/EllipsisMenu/EllipsisMenu'
+import { useTranslation } from 'contexts/Localization'
+import { usePriceCakeBusd } from 'state/farms/hooks'
+import { Auction, Bidder } from 'config/constants/types'
+import WhitelistedBiddersModal from '../WhitelistedBiddersModal'
+
+const LeaderboardContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 5fr 3fr 1fr;
+  ${({ theme }) => theme.mediaQueries.md} {
+    grid-template-columns: 3fr 5fr 5fr 1fr;
+  }
+`
+
+const GridCell = styled(Flex)<{ isTopPosition }>`
+  height: 65px;
+  align-items: center;
+  border-bottom: 1px solid ${({ theme }) => theme.colors.cardBorder};
+
+  ${({ theme, isTopPosition }) =>
+    isTopPosition &&
+    `
+    background-color: ${theme.colors.warning}2D;
+  `}
+`
+
+const BidderEllipsisMenu: React.FC<{ bidder: Bidder }> = ({ bidder }) => {
+  const { t } = useTranslation()
+  const handleSortOptionChange = (option: OptionProps) => {
+    window.open(option.value, '_blank').focus()
```

Comment:
can you use a Link component instead ?

Reply:
Fixed in 8da88d230394d6bd5e876dcd1725bd4aca937ed7